valhalla/sif/dynamiccost.h

```diff
@@ -240,7 +240,7 @@ class DynamicCost {
         ((disallow_mask & kDisallowStartRestriction) && edge->start_restriction()) ||
         ((disallow_mask & kDisallowEndRestriction) && edge->end_restriction()) ||
         ((disallow_mask & kDisallowSimpleRestriction) && edge->restrictions());
-    return !edge->is_shortcut() && accessible && !assumed_restricted;
```

Comment:
you know what might have been more elegant, adding a new option to the disallow mask for shortcuts. what do you think about that @nilsnolde? if you dont have the time but you like the idea i could take a whack at it too. let me know

Reply:
@nilsnolde that is a big feature request, we should write up an issue for it imho and start a larger discussion before we jfdi, is that ok with you? i can write one up, but i feel like others have asked for this before too...

Reply:
was hoping for less big. sure, that'd be nice @kevinkreiser

Reply:
sadly "one does not simply" swap out the tilecache in a multithreaded process where its shared across threads.. thats the main eventuality that we'd have to decide how we feel would be ok to handle it

Reply:
in any case, I switched to bit masking for shortcuts as well